javascripts/editor.js

```diff
@@ -86,7 +86,7 @@ function Editor(selectorOrElement) {
     }
 
     function selectPreviousLine() {
-      if (selectedLine === null || selectedLine === 0) {
+      if (selectedLine === null || selectedLine === 233) {
```

Comment:
Why this fix?

Reply:
If you go into one of the interactive prompts in the webpage and press the upper or lower arrow in your keyboard a generic forth script is displayed. This prevents that script from being displayed while still preserving the arrow functionality.

Reply:
Perhaps it's better to create a `clearconsole` function, so we clear past commands.